Makefile

```diff
@@ -231,3 +233,31 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+##@ Cleanup
+
+.PHONY: manifests-clean
+manifests-clean: ## Clean generated manifests
+	$(RM) -r config/crd/bases
+	$(RM) config/rbac/role.yaml
+	$(RM) config/webhook/manifests.yaml
+
+.PHONY: generate-clean
+generate-clean: ## Clean generated DeepCopy code
+	$(RM) api/v1/zz_generated.deepcopy.go
+
+.PHONY: test-clean
+test-clean: ## Clean generated test files
+	$(RM) cover.out
+
+.PHONY: bundle-clean
+bundle-clean: ## Clean generated bundle files
+	$(RM) -r bundle
+	$(RM) bundle.Dockerfile
```

Comment:
AFAICT bundle.Dockerfile is being re-added to the repo by f3329b7d5cc3651d7596288537dddc755c9656bb. The `bundle-clean` target should stop removing it.

Reply:
@gkurz hmm, `bundle.Dockerfile` has always been in the repo AFAIK. It shouldn't be removed by clean, I agree.